scripts/pre_commit_linter.py

```diff
@@ -729,10 +729,15 @@ def _pre_commit_linter(all_files):
         process.daemon = False
         process.start()
 
+    number_of_files_to_lint = (
+        len(js_files_to_lint) + len(py_files_to_lint) +
+        len(html_files_to_lint_for_css) + len(css_files_to_lint))
```

Comment:
Thanks, this looks great! Maybe we should also divide by `len(linting_processes)`, though? Also, what is the value of timeout_for_lint_process right now?

Reply:
The number of files for the complete lint check is 1640, so the timeout comes to be around 650 ms. Please feel free to suggest any other proportion.
> we should also divide by len(linting_processes)

Oh, so each lint process should have a different timeout based on the number of files? That sounds nice!

Reply:
Yup, I think dividing by it alone should hopefully give good enough of a proportion?

Reply:
Done, ptal!